src/bionics_ui.cpp

```diff
@@ -36,6 +37,94 @@ enum bionic_menu_mode {
     EXAMINING,
     REASSIGNING
 };
+
+enum bionic_sort_mode {
+    POWER,
+    NAME,
+};
+bionic_sort_mode _bionic_sort_mode = bionic_sort_mode::POWER;
+
+auto sort_mode_str( bionic_sort_mode mode ) -> std::string
```

Comment:
```suggestion
std::string sort_mode_str( bionic_sort_mode mode )
```
We don't currently use the "`auto` first" pattern for functions in CDDA (unless the return type is really long).

There are a few more functions you've added that need changing similarly.

Reply:
Sorry, habit. I'll fix them

Reply:
I'm not exactly convinced by the "we don't currently use..." argument, but I'll match the style for now. There's no reason we can't use modern patterns for new code